CEPCPreamble.md

```diff
@@ -0,0 +1,40 @@
+
+3. Think of others’ needs from their point of view. Use preferred names, titles (including pronouns), and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others and acknowledge the contributions of your peers. 
+It is impossible to be aware of all of someone else's needs without asking them what they need. This can be as simple as how to pronounce a name or more subtle, like recognizing that some people prefer to contribute by listening. Avoid making assumptions others' preferences and needs. When in doubt, ask, and respect the response. It's important to recognize the contributions of others. 
+
+Unaccetpable behaviors include:
+4. Offensive comments related to socio-economic status, sexual orientation, religion, race, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
+It is never acceptable to say something offensive. You might not realize that your comments are offensive. This code priorotizes impact of statements over intent. You might not have intended a comment to be offensive, but it can still be hurtful. In such a situation, an apology or correction is still be called for.
+
+5. Misgendering someone by deliberately referring to a person using the wrong pronouns or by using someone's proper names or other terms that person has asked not to be used, also known as deadnaming.
+When someone provides a name to refer to them by, respect the information they provide about themselves even if it is different from information with which you are familiar. Someone may have gone through a recent life change that brought about a name change. Someone may have recently married, divorced, or transitioned. 
+
+6. Stalking or physically following or invading someone's personal space after a request to stop.
+
+Respect people's personal space, especially if they've expressed a preference for a boundary. Different cultures and individuals have different tolerances for what is considered an appropriate amount of space when talking. Some people prefer to avoid any touch. So much more so, do not follow others. Stalking is an extreme form of this that is certainly not acceptable.
+
+7. Microaggressions, which are small comments or questions, either intentional or unintentional, that marginalize people by communicating hostile, derogatory, or negative beliefs. Be aware that, regardless of the speaker's intentions, some phrases or constructions lead people to expect a patronizing statement to follow, and avoid such phrases. For example, beginning an interjection with "Well, actually..." can set this expectation and be taken as a sign of disrespect. and Feigning surprise at someone’s lack of knowledge or awareness about a topic.

```

Comment:
I don't think "Well, actually..." is a great example of microaggression, because: 1. It is complex.  I don't think it is universally disrespectful; rather it is sometimes used disrespectfully, and for some listeners it is a trigger term that is taken as indicating disrespect (whether or not any disrespect was intended).  2. I don't think it is easily understandable as an example.  I think any example that is not easily understood (on face value) should be explained, but this doesn't seem like the right context for explaining examples.   I think feigning surprise is a good example here.